pages/dashboard/v2/details/index.js

```diff
@@ -0,0 +1,30 @@
+import Navbar from '../../../../components/navbar';
+import Layout from '../../../../components/layout';
+import Head from 'next/head';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+
+export default function Details() {
+  const router = useRouter();
+  const data = router.query;
+  return (
+    <Layout>
+      <Head>
+        <title>Student Details</title>
+        <meta name='description' content='Generated by create next app' />
+        <link rel='icon' href='/favicon.ico' />
+      </Head>
+      <Navbar>
+        <div className='border-solid border-2 pl-4 pr-4'>
+          <Link href={'/classes'}>Classes</Link>
+        </div>
+        <div className='border-solid border-2 pl-4 pr-4'>
+          <Link href={'/'}> Menu</Link>
+        </div>
+      </Navbar>
+      <div>
+        <h1>{`${data.studentName}'s Progress in: ${data.classroomName} `}</h1>
+      </div>
+    </Layout>
+  );
+}
```

Comment:
@ngillux @Komal914
FWIW, this comment is to test GitHub's feature: Suggested change
```suggestion
import Head from 'next/head';
import Navbar from '../../../../../../components/navbar';
import Link from 'next/link';
import { getSession } from 'next-auth/react';
import { useRouter } from 'next/router';

export async function getServerSideProps(context) {
const userSession = await getSession(context);
if (!userSession) {
context.res.writeHead(302, { Location: '/' });
context.res.end();
return {};
}
return {
props: {
userSession
}
};
}

export default function Details({
userSession
}) {
const router = useRouter();
const data = router.query;
const studentName = data.studentName;
const classroomName = data.classroomName;
return (
<>

<title>Create Next App</title>



{userSession && (
<>


<Link href={'/classes'}>Classes


<Link href={'/'}> Menu


{studentName}' progress in {classroomName}

</>
)}
</>
);
}
```